DeAceroDLabs/marketplace-frontend
# [DMP-430] Login view

## What is done

[[DMP-430] Login View](https://deacero.atlassian.net/browse/DMP-430)

## How is done

1. Created the Login UI
2. Created a component for rendering forms dynamically from a json structure.
3. Created a Field component, which will check for the type of component that needs to be rendered in the dynamic form.
4. Created the TextField component, which will render an input of type text.

## How to test

1. Run the project via `npm start`
2. You will be redirected to the Login View. Check that it matches the mockups.
3. Enter your credentials (it doesn't really validates them, so it can be anything).
4. Click on the login button.
5. You should be redirected to the home.

## Evidence

Add images or screen recordings on how the feature was working after and before the fix (if applies)
